README.md

```diff
@@ -77,6 +78,7 @@ Examples:
     goose mssql "sqlserver://user:password@dbname:1433?database=master" status
     goose clickhouse "tcp://127.0.0.1:9000" status
     goose vertica "vertica://user:password@localhost:5433/dbname?connection_load_balance=1" status
+    goose ydb "grpcs://localhost:2135/local" status
```

Comment:
I've never used ydb, so know very little about it.

Does this mean the connection is over a secure gRPC connection? I might go snooping around the driver...

Reply:
Yes, it secure gRPC.